CA_DataUploader/Program.cs

```diff
@@ -18,10 +19,11 @@ static async Task MainAsync(string[] args)
         {
             try
             {
-                CALog.LogInfoAndConsoleLn(LogID.A, RpiVersion.GetWelcomeMessage($"Upload temperature data to cloud"));
+                var loglevel = IOconfFileLoader.FileExists() ? IOconfFile.Instance.GetOutputLevel() : IOconfLoopName.Default.LogLevel;
+                CALog.LogInfoAndConsoleLn(LogID.A, RpiVersion.GetWelcomeMessage($"Upload temperature data to cloud", loglevel));
                 Console.WriteLine("Initializing...");
                 Redundancy.RegisterSystemExtensions();
-                var serial = new SerialNumberMapper();
+                var serial = new SerialNumberMapper(null);

```

Comment:
I think not passing a loaded ioconf would give trouble if answering yes to "Do you want to skip IO.conf setup? (yes, no)" in IOconfSetup.UpdateIOconf (answering no causes config to be updated and boards to be mapped again during reload).

Reply:
fixed